examples/stable/test/property/ic_api/call/src/callee.ts

```diff
@@ -0,0 +1,22 @@
+import { IDL, Principal, query, update } from 'azle';
+
+// A simple canister that provides methods for testing inter-canister call errors
+export default class {
+    @query([], IDL.Principal)
+    getSelf(): Principal {
+        return ic.id();
+    }
+
+    // This method will explicitly reject the call
+    @update([], IDL.Bool)
+    explicitly_reject(): boolean {
+        ic.reject('Explicitly rejected for testing');
+        return true; // This will never be reached
```

Comment:
Where is this ic object coming from? Did you even run these tests? I don't see how these could have passed. Reject should be imported from azle as msgReject